HEN_HOUSE/omega/progs/addphsp/Makefile

```diff
@@ -45,3 +45,9 @@ addphsp_$(my_machine).f: $(SOURCES)
 	@echo "Mortran compiling addphsp sources ..."
 	@$(MORTRAN_EXE) -d $(MORTRAN_DATA) -f $(SOURCES) -o7 $@ \
 	    -o8 addphsp_$(my_machine).mortlst
+
+clean:
+	$(REMOVE) mortjob.mortran addphsp_$(my_machine).F addphsp_$(my_machine).mortlst
```

Comment:
@blakewalters this should be `addphsp_$(my_machine).f` (lowercase `f`) I think, according to compilation target above.